### DISCUSSION TITLE
YPbPr showing black and white

### DISCUSSION THREAD

**Discussion by mike-uk** [GerryDazoo/Slinger#578] [Q&A]
Got a new set-top box setup on Pro-HD remotely, the yPbPr is showing black and white image the colours cable seem to plug in correctly. Can it be 1080p vs 1080i issue? or the producer mixed up the colour so how?

**Comment by blasco403** [comment]
Usually this is caused by a bad component cable. I would try a different cable set.

**Reply by mike-uk** [reply]
The cable is the one came with the Pro-HD when it was new and never used. It is like 10 years old though.
